README.md

```diff
@@ -26,18 +26,18 @@ cd open-source-edu-bot
 python3 -m venv venv
 ```
 
-on Windows
+On Windows
 
 ```PowerShell
 py -3 -m venv venv
 ```
 
 **Activate the environment**
 
-Copy the environments file and adapt it:
+Rename the environments file and adapt it:
```

Comment:
Do not `rename` but `copy`, revert this change

Reply:
I am the one who asked for the rename suggestion! Only copying is not clear for non-*unix users! @vj-codes I think renaming alone is also a bit misleading! so probably we combine both, copy and rename 👍

Reply:
Actually the file `.sample.env` should not be `modified`, so not `renamed`. If the admin mess up with the config he can always restart by copying it.